css/platform.css

```diff
@@ -0,0 +1,41 @@
+/*Each row of the grid is turned into a table row (<tr> element). The strings in
+the grid are used as class names for the table cell (<td>) elements.*/
+
+.background {
+  background: rgb(52, 166, 251);
```

Comment:
[Google's style guide](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) encourages us to alphabetize our declarations.